services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -604,10 +604,8 @@ async def manage_get_response(  # pylint: disable=too-many-locals, too-many-bran
         ):
             detail = {"error": "Unauthorized access."}
             return JSONResponse(content=detail, status_code=HTTP_401_UNAUTHORIZED)
-        elif "/queryables" in request.scope["path"]:
-            content["$id"] = request.url._url  # pylint: disable=protected-access
         elif (
-            "/collections" in request.scope["path"] and "items" not in request.scope["path"]
+            "/collection" in request.scope["path"] and "items" not in request.scope["path"]
```

Comment:
Typo check:
previously it was collection**s**

Reply:
Solved